tool/tsh/common/tsh.go

```diff
@@ -1033,12 +1035,14 @@ func Run(ctx context.Context, args []string, opts ...CliOption) error {
 	reqCreate.Flag("request-ttl", "Expiration time for the access request").DurationVar(&cf.RequestTTL)
 	reqCreate.Flag("session-ttl", "Expiration time for the elevated certificate").DurationVar(&cf.SessionTTL)
 	reqCreate.Flag("max-duration", "How long the the access should be granted for").DurationVar(&cf.MaxDuration)
+	reqCreate.Flag("assume-time", "Sets time roles can be assumed by requestor (RFC3339)").StringVar(&cf.AssumeTimeRaw)
```

Comment:
I also have concerns that users will have a hard "time" inputting the correct format. For example, if I use the date from `date --rfc-3339=seconds`, it will fail.

Reply:
Would you suggest swapping to a different time format that is a bit friendlier or should I add an example time  to the help string so users can see the format without having to look it up?